app/services/push_notification/recovery_activated_event.rb

```diff
@@ -0,0 +1,30 @@
+module PushNotification
+  class RecoveryActivatedEvent
+    EVENT_TYPE = 'https://schemas.openid.net/secevent/risc/event-type/recovery-activated'.freeze
+
+    attr_reader :user
```

Comment:
is `user` only used for comparison in specs?

Reply:
the push sender reads the user property off the event (it makes the sending interface easy, `HttpPush.deliver(Event.new(user))` instead of duplicating, ex `HttpPush.deliver(user, Event.new(user))` because at least one event needs the user object for its own data)